horace_core/sqw/file_io/@sqw_binfile_common/get_data.m

```diff
@@ -1,4 +1,4 @@
-function [data,obj] = get_data (obj,varargin)
+function [data,obj] = get_data(obj,varargin)
```

Comment:
new page-size argument is missing from the function header

Reply:
@nickbattam-tessella it looks as if it is in varargin and processed by parse_args as a keyword argument